RepresentsMe.xcodeproj/project.pbxproj

```diff
@@ -7,6 +7,16 @@
 	objects = {
 
 /* Begin PBXBuildFile section */
+		13287BB92220479B00622CD4 /* OfficialScraperTests.swift in Sources */ = {isa = PBXBuildFile; fileRef = 13287BB82220479B00622CD4 /* OfficialScraperTests.swift */; };
+		13287BBB2220490E00622CD4 /* apikey.swift in Sources */ = {isa = PBXBuildFile; fileRef = 13287BBA2220490E00622CD4 /* apikey.swift */; };
```

Comment:
Yeah this is what I'm worried about

Reply:
Deemed good based on offline conversation. @tirtow will move apikey.swift to main folder and organize API scraping files into folder(s)

Reply:
86e1d35 resolves this.